### DISCUSSION TITLE
The cursor disappears when F11 is pressed

### DISCUSSION THREAD

**Discussion by Koushikphy** [monkeytypegame/monkeytype#4028] [Questions] [answered]
The cursor disappears when you press F11 to toggle in and out of full screen. The cursor comes back when a new test is started. Is it a problem with my browser or really a bug?

**Comment by Miodec** [comment, marked as answer]
Next time open an actual bug report in the issues tab - I might miss discussions.

Either way, this issue should be fixed by now.
